tests/lib/validator_test.py

```diff
@@ -548,3 +554,91 @@ def _create_route(dest, via_addr, via_iface, table, metric):
         schema.Route.NEXT_HOP_INTERFACE: via_iface,
         schema.Route.TABLE_ID: table,
     }
+
+
+class TestLinuxBondValidator:
+    def test_mode_not_defined(self):
+        iface_state = self._gen_iface_state("bond0")
+        iface_state[Bond.CONFIG_SUBTREE].pop(Bond.MODE)
+        original_desired_state = state.State({Interface.KEY: [iface_state]})
+        desired_state = state.State({Interface.KEY: [iface_state]})
+        current_state = empty_state()
+        with pytest.raises(NmstateValueError):
+            libnmstate.validator.validate_interfaces_state(
+                original_desired_state, desired_state, current_state
+            )
+
+    def test_mac_restriction_without_mac_in_desire(self):
```

Comment:
IMHO "mac_restriction" should not be part of the test name, but fail_over_active how about:

`test_fail_over_active_without_mac_in_desired`? This applies to the other tests as well.

Reply:
It's `fail_over_mac_active_with_active_back_mode`. Just too long and meaningless.